config/database.yml

```diff
@@ -1,15 +1,15 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  username: <%= ENV.fetch("POSTGRES_USER") { } %>
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   port: <%= ENV.fetch("POSTGRES_PORT", "5432") %>
-  password: <%= ENV.fetch("POSTGRES_PASSWORD") { 'password' } %>
 
 development:
   <<: *default
   database: casa_development
   host: <%= ENV.fetch("DATABASE_HOST") { } %>
+  username: <%= ENV.fetch("POSTGRES_USER") { } %>
+  password: <%= ENV.fetch("POSTGRES_PASSWORD") { 'password' } %>
```

Comment:
Moving out of default is gonna disrupt tests. Any reason why you're moving it from the default?
<img width="1130" alt="Screenshot 2024-06-01 at 6 40 35 AM" src="https://github.com/rubyforgood/casa/assets/50247514/07125a4a-fdcd-4c78-a13f-60dff95e37e1">

Reply:
It was causing issues on some macs. IIRC they needed to have nothing set for poth PGUSER and PASSWORD and doing `PGUSER=` did not work.

Reply:
This is what my `.env` file looks like. Maybe a callout to create this and what to set/try in the docs?
<img width="375" alt="Screenshot 2024-06-01 at 7 41 19 AM" src="https://github.com/rubyforgood/casa/assets/50247514/d9d086f3-4b2f-4dc8-9034-ca03f62fb70e">

Reply:
Yea, I think this may not be the best solution as I am learning every environment is vastly different.  To @schoork , it may be better to point this out in the docs when setting up a new environment.